### PR TITLE
Update to 1.19

### PR DESCRIPTION
**Pull request by js802025**
Makes mod work in 1.19/

### CODE REVIEW

**Comment by kosma** [comment]
Thanks! Looks good. Only thing that needs changing is, please don't commit `.vscode` and `bin`, these are not needed.

**Comment by js802025** [comment]
ok. i will fix.

**Comment by js802025** [comment]
should be fixed.

File: src/main/java/pl/kosma/geodesy/GeodesyCore.java [hunk 1]
```diff
@@ -304,7 +303,7 @@ private void highlightGeode() {
             LOGGER.error("StructureBlock tile entity is missing... this should never happen????");
             return;
         }
-        structure.setStructureName("geode");
+        //structure.setStructureName("geode");
```
Comment (kosma) [inline src/main/java/pl/kosma/geodesy/GeodesyCore.java:306]: bruh

**Comment by kosma** [comment]
Merged in https://github.com/kosma/geodesy-fabric/commit/7b29b950e2726ba37d576a1079be1b92969feca6, I have fixed a few other things.

**Comment by kosma** [comment]
Also: thanks!

**Pull request closed without merging**